var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -89,9 +91,26 @@ class Trilinos(CMakePackage):
             description='Builds a debug version of the libraries')
     variant('boost',        default=True,  description='Compile with Boost')
     variant('tpetra',       default=True,  description='Compile with Tpetra')
-    variant('exodus',       default=False, description='Compile with Exodus from SEACAS')
-
-    variant('dtk',          default=False, description='Enable DataTransferKit')
+    variant('epetra',       default=True,  description='Compile with Epetra')
```

Comment:
nevermind this comment, we would still do stuff manually below to turn it on/off, so don't bother with that automation.